### PR TITLE
Game page top section

### PR DESCRIPTION
**Pull request by utrennik**
Capture

*** Game page TOP section ***
* should contain game title (uneditable)
* should contain the dealer user card with name and avatar
* hsould contain stop game (in DEALER mode) or exit game (in USER mode)
* should contain working downtimer with reset function
* should contain run round button (if is not running) or reset round (if the game is running)

Out of scope:
* buttons are doing nothing

### CODE REVIEW

File: client/src/components/pages/game-page/game-top-section/game-timer/game-timer.sass [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@import '@styles/variables'
+
+.timer-body
+  font-size: 40px
+  font-weight: 700
+  color: #fff
```
Comment (aksiantsiuk) [inline client/src/components/pages/game-page/game-top-section/game-timer/game-timer.sass]: use color from variables or add a new one

**Review by HelenBassa — APPROVED** [review]

File: client/src/components/pages/game-page/game-top-section/game-top-section.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <div className="game-top-section">
+      <div className="game-top-section-title">{gameTitle}</div>
+      <div className="game-top-section-row">
+        <div className="game-top-section-dealer game-top-col">
+          <ScramMasterCard
+            firstName={firstName}
+            lastName={lastName}
+            role={jobPosition || ''}
+            avatarImage={avatar}
+            isScramMasterLobby={client && client.id === id}
```
Comment (aksiantsiuk) [inline client/src/components/pages/game-page/game-top-section/game-top-section.tsx]: make a separate understantable variable like `isScramMasterLobby = client && client.id === id`

File: client/src/components/pages/game-page/game-top-section/game-top-section.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          />
+        </div>
+
+        <div className="game-top-section-timer game-top-col">
+          <GameTimer timerSecs={timerSecs} />
+        </div>
+
+        <div className="game-top-section-btn game-top-col">
+          <Button variant="contained" size="medium" color="primary">
+            {client.clientUser && client.clientUser.role === config.DEALER
```
Comment (aksiantsiuk) [inline client/src/components/pages/game-page/game-top-section/game-top-section.tsx]: make a separate understantable variable like `isDealer =client.clientUser && client.clientUser.role === config.DEALER `

File: client/src/components/pages/game-page/game-top-section/game-timer/game-timer.sass (hunk 1 above)
Comment (utrennik) [reply client/src/components/pages/game-page/game-top-section/game-timer/game-timer.sass]: Fixed

File: client/src/components/pages/game-page/game-top-section/game-top-section.tsx (hunk 2 above)
Comment (utrennik) [reply client/src/components/pages/game-page/game-top-section/game-top-section.tsx]: Fixed

File: client/src/components/pages/game-page/game-top-section/game-top-section.tsx (hunk 3 above)
Comment (utrennik) [reply client/src/components/pages/game-page/game-top-section/game-top-section.tsx]: Fixed

**Review by RealOneReal — APPROVED** [review]

**Pull request merged**